test/alerts/alerts.js

```diff
@@ -48,113 +48,121 @@ async function cleanup() {
     return;
 }
 
-afterEach(async () => {
-    sinon.restore();
-    await cleanup();
-});
```

Comment:
This `afterEach` was outside of any descibe, so it was running after **every test**.  This is what was causing the "after hook" error messages.